src/subscription_manager/plugins.py

```diff
@@ -438,6 +438,26 @@ def __init__(self, clazz, filter_string, installed_products):
         self.installed_products = installed_products
 
 
+class PreUnRegisterConduit(BaseConduit):
```

Comment:
OK. I will not add more plugin hooks. I swear :-). Are we going to remove plugin framework completely somewhere in the future?